sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -128,6 +128,54 @@ def __init__(self, body, **kwargs):
     def __str__(self):
         return str(self.message)
 
+    def __repr__(self):
+        # type: () -> str
+        # pylint: disable=bare-except
+        message_repr = "body={}".format(
+            str(self.body)
```

Comment:
I remember `self.body` is returning a generator -- `str(self.body)` would print `<generator object DataBody.data at 0x000001F967066A48>`, `self.message` should be good
```suggestion
            str(self.message)
```